clients/src/test/java/org/apache/kafka/test/MetricsBench.java

```diff
@@ -37,7 +37,7 @@ public static void main(String[] args) {
             Sensor child = metrics.sensor("child", parent);
             for (Sensor sensor : Arrays.asList(parent, child)) {
                 sensor.add(metrics.metricName(sensor.name() + ".avg", "grp1"), new Avg());
-                sensor.add(metrics.metricName(sensor.name() + ".count", "grp1"), new Count());
+                sensor.add(metrics.metricName(sensor.name() + ".count", "grp1"), new WindowedCount());
```

Comment:
nit: also change the metric name?

Reply:
I don't know... I'm reluctant to expand the scope of this change to change registered metric names. I know these are just in tests, but still, it seems not necessary.

Do you think changing these metric names increases clarity, or were you just asking if we should consider changing them?

Reply:
I do not feel strong about it, just asking whether it's worth doing. Seems it's not worth doing after all.

Reply:
Ok. Thanks for bringing it up, but let's just leave it alone... It will be an opportunity for someone to use `git blame` later and curse my name.